lib/Backend/GlobOpt.cpp

```diff
@@ -19058,6 +19058,44 @@ GlobOpt::OptDstIsInvariant(IR::RegOpnd *dst)
 }
 
 void
+GlobOpt::OptHoistToLandingPadUpdateValueType(
+    BasicBlock* landingPad,
+    IR::Instr* instr,
+    IR::Opnd* opnd,
+    Value* opndVal)
+{
+    if (instr->m_opcode == Js::OpCode::FromVar)
```

Comment:
I don't see the source of the copy, where is it?

---
In reply to: [101616597](https://github.com/Microsoft/ChakraCore/pull/2538#discussion_r101616597) [](ancestors = 101616597,101593051,101556159)